pints/_mcmc/__init__.py

```diff
@@ -1102,3 +1102,76 @@ def mcmc_sample(log_pdf, chains, x0, sigma0=None, method=None):
     """
     return MCMCController(    # pragma: no cover
         log_pdf, chains, x0, sigma0, method=method).run()
+
+
+def initialise_finite(log_pdf, chains, random_sampler=None, max_n_tries=50,
```

Comment:
- Could we get a nicer name for this? E.g. `sample_initial_points` ?
- Could `chains` just be `n`, so that it's more general?
- If so, then do we want this for other (non-mcmc) methods as well?

Reply:
Great points @MichaelClerx -- I've changed it to `sample_initial_points`. I've changed `chains` to `n_points`.